wallets/index.js

```diff
@@ -223,8 +226,11 @@ export function useWallet (name) {
 }
 
 export function useConfiguredWallets () {
-  const { wallets } = useWallets()
-  return useMemo(() => wallets.filter(w => isConfigured(w)), [wallets])
+  const { wallets, loading } = useWallets()
+  return useMemo(() => ({
+    wallets: wallets.filter(w => isConfigured(w)),
+    loading
+  }), [wallets, loading])
 }
```

Comment:
Oh, putting this back into draft because I forgot to check if we use `useConfiguredWallets` in other places, too

Reply:
Ok, we don't.